content/en/docs/queue_resource_management.md

```diff
@@ -0,0 +1,142 @@
++++
+title = "Queue Resource Management"
+
+date = 2024-12-30
+lastmod = 2024-12-30
+
+draft = false  # Is this a draft? true/false
+toc = true  # Show table of contents? true/false
+type = "docs"  # Do not modify.
+
+# Add menu entry to sidebar.
+[menu.docs]
+  parent = "features"
+  weight = 1
++++
+
+## Overview
+
+[Queue](/en/docs/queue) is one of the core concepts in Volcano, used to support resource allocation and task scheduling in multi-tenant scenarios. Through queues, users can achieve multi-tenant resource allocation, task priority control, resource preemption and reclamation, significantly improving cluster resource utilization and task scheduling efficiency.
+
+### Core Features
+
+1. Flexible Resource Configuration
+
+   * Support multi-dimensional resource quota control (CPU, Memory, GPU, NPU, etc.)
+   * Provide three-level resource configuration mechanism:
+      * Capability: Upper limit of queue resource usage
+      * Deserved: Expected resource amount (When no other queues submit jobs, the resources occupied by jobs in this queue can exceed the Deserved value. When multiple queues submit jobs and cluster resources are insufficient, resources exceeding the Deserved value can be reclaimed by other queues)
+      * Guarantee: Reserved resources (Reserved resources can only be used by this queue, other queues cannot use them)
+
+      > Recommendations: 
+      > 1. When configuring three levels of resources, follow: Guarantee <= Deserved <= Capability;
+      > 2. Guarantee can be configured as needed, Deserved value must be configured when enabling the Capacity plugin;
+      > 3. Deserved configuration suggestions: In peer queue scenarios, the sum of Deserved values for all queues equals the total cluster resources; In hierarchical queue scenarios, the sum of child queues' Deserved values equals their parent queue's Deserved value.
+
+   * Support dynamic resource quota adjustment
+
+2. [Hierarchical Queue](/en/docs/hierarchical_queue) Management
+
+   * Support multi-level queue structure
+   * Provide resource inheritance and isolation between parent and child queues
+   * Compatible with Yarn-style resource management mode for big data workload migration
+   * Support cross-level queue resource sharing and reclamation
+
+3. Intelligent Resource Scheduling
+
+   * Resource borrowing: Allow queues to use idle resources from other queues
+   * Resource reclamation: Prioritize reclaiming excess resources when resources are tight
+   * Resource preemption: Ensure resource requirements for high-priority tasks
+
+4. Multi-tenant Isolation
+
+   * Strict resource quota control
+   * Priority-based resource allocation
+   * Prevent single tenant from excessive resource usage
+
+## Queue Scheduling Implementation
+### Related Scheduling Actions
```

Comment:
We should emphasize 
it's queue related actions.

Reply:
![image](https://github.com/user-attachments/assets/a8827b6b-4f9b-4030-a6de-438d46862e28)
updated